packages/design-system/src/components/Table/Table.tsx

```diff
@@ -58,27 +121,31 @@ export class Table extends React.PureComponent {
     }
   }
 
-  componentDidMount() {
+  componentDidMount(): void {
     if (this.props.scrollable) {
       window.addEventListener('resize', this.debounceHandleResize);
       this.handleResize();
     }
   }
 
-  componentWillUnmount() {
+  componentWillUnmount(): void {
     if (this.props.scrollable) {
       window.removeEventListener('resize', this.debounceHandleResize);
     }
   }
 
-  handleResize() {
+  captionID: string;
+  container: any;
+  debounceHandleResize: (...args: any[]) => any;
+
+  handleResize(): void {
     const { scrollWidth, clientWidth } = this.container;
     const scrollActive = scrollWidth > clientWidth;
     this.setState({ scrollActive });
   }
 
-  renderChildren() {
-    return React.Children.map(this.props.children, (child) => {
+  renderChildren(): React.ReactNode[] {
+    return React.Children.map(this.props.children, (child: React.ReactElement) => {
```

Comment:
Referring to https://github.com/CMSgov/design-system/pull/1088/files#r673582273, it's somewhere in this block where I start getting errors if I try to optionally render any child of `Table`.